datadog_checks_downloader/pyproject.toml

```diff
@@ -19,6 +19,7 @@ authors = [
     { name = "Datadog", email = "packages@datadoghq.com" },
 ]
 classifiers = [
+    "Private :: Do Not Upload",
```

Comment:
This is a good idea, although maybe we should also do this for all of the integrations excluding the base check, since we don't want them on PyPI either. Maybe a script can be used to apply this classifier? @ofek thoughts?